check_stability.py

```diff
@@ -107,6 +106,48 @@ def __exit__(self, type, value, traceback):
         print("travis_fold:end:%s" % self.name, file=sys.stderr)
 
 
+class FilteredIO(StringIO):
```

Comment:
I've been researching how to do this more completely, and I found the following
documentation:

> `stdout` and `stderr` needn’t be built-in file objects: any object is
> acceptable as long as it has a `write()` method that takes a string argument.

(https://docs.python.org/2/library/sys.html#sys.stdout)

It seems like for this use case, the naive "write-only" implementation is
sufficient. Would you agree?